### PR TITLE
Collapsible: allow dynamic height

### PR DESCRIPTION
**Pull request by dan-lee**
Checklist:

- [x] Test added / Snapshots updated
- [ ] Documentation added / updated

### CODE REVIEW

File: src/molecules/Collapsible.jsx [hunk 1]
```diff
@@ -212,10 +224,11 @@ class Collapsible extends React.Component {
         {/* Child */}
         <View
           onRef={this.childRef}
+          onTransitionEnd={this.handleTransitionEnd}
```
Comment (yamafaktory) [inline src/molecules/Collapsible.jsx]: Neat!

**Comment by yamafaktory** [comment]
Well, this a way simpler and robust implementation performance-wise than #288! Closes #288.

**Comment by yamafaktory** [comment]
@dan-lee @mosch I would like to reopen that PR or at least we should try to find an alternative solution for the dynamic height of the content. This is still an issue with the current implementation.

**Comment by dan-lee** [comment]
@yamafaktory Sure, what's the issue? How to reproduce? I'd like to find a pure CSS fix if possible.

**Comment by yamafaktory** [comment]
@dan-lee you can have a look at https://github.com/allthings/cockpit/pull/963. If you reach the detail page of any user (user management) and then, for a given channel, add more roles till those are overlapping the content of the collapsible. The collapsible component is not dynamically updating his height to its content.

**Comment by dan-lee** [comment]
I don't think this PR is released yet. I can see, that the height is not changing to `auto` after collapsing/expanding.

**Pull request merged**